I can't get the problem:
`FAILED test/test_build_caches.py::test_build_caches - RuntimeError: humble: missing package.xml file for package "fluent_bit_vendor"

It is here https://github.com/Minipada/fluent_bit_vendor/blob/humble/package.xml#L4

--- a/humble/distribution.yaml
+++ b/humble/distribution.yaml
@@ -1360,11 +1360,6 @@ repositories:
       version: humble-devel
     status: maintained
   fluent_bit_vendor:
-    release:
-      tags:
-        release: release/humble/{package}/{version}
-      url: https://github.com/minipada/fluent_bit_vendor-release.git
-      version: 2.0.0
     source:
       type: git
       url: https://github.com/minipada/fluent_bit_vendor.git

